### PR TITLE
{Release} Upgrade to Azure CLI 2.9.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,137 @@
 Release History
 ===============
 
+2.9.0
+++++++
+
+**1**
+
+* Addition of az security iot-solution/ iot-alerts/iot-recommendations/iot-analytics REST to Azure CLI (#14124)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:11]: The `**1**` is caused by https://github.com/Azure/azure-cli/pull/14124

> [1] addition of az security iot-solution/ iot-alerts/iot-recommendations/iot-analytics REST to Azure CLI

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,137 @@
...
+
+**Compute**
+
+* Bump version disks 2020-05-01, compute 2020-06-01 (#14212)
+* Double encryption of disk encryption set (#14212)
+* `az vmss update`: support specify cross tenant image. (#14206)
+* `az sig image-version create`: support specify cross tenant image. (#14206)
+* vm/vmss create: Encryption of cache & data-in-transit for OS/Data disks and temp disks for VM & VMSS (#13919)
+
+**compute**
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:45]:
```suggestion
**Compute**
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,137 @@
...
+
+**security**
+
+* Add regulatory compliance CLI (#14103)
+
+**Security**
+
+* Add allowed_connections commands (#14190)
+* Add Adaptive network hardeningss commands (#14260)
+* Add adaptive_application_controls commands (#14278)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:105]: These need to be unified.

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,137 @@
...
+
+* `az storage account create/update`: Add --allow-blob-public-access to allow or disallow public access for blob and containers (#13986)
+* `az storage account create/update`: Add `--min-tls-version` to support setting the minimum TLS version to be permitted on requests to storage. (#14131)
+* Remove check in token credential (#14134)
+* Fix the storage account name in examples (#14062)
+
+**Thoth**
+
+* : recommend_recovery_options: version: "2.8.0", command: "account", parameters: "['--debug']", extension: "None" (#14180)
+* : User must agree to telemetry before failure recovery recommendations can be presented. (#14180)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:127]: @fengzhou-msft. This is caused by an incorrect history extraction. https://github.com/Azure/azure-cli/pull/14180 has these lines as code:

```
[Thoth]: recommend_recovery_options: version: "2.8.0", command: "account", parameters: "['--debug']", extension: "None"
[Thoth]: User must agree to telemetry before failure recovery recommendations can be presented.
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,134 @@
...
+**security**
+
+* Add regulatory compliance CLI (#14103)
+
+**Security**
+
+* Add allowed_connections commands (#14190)
+* Add Adaptive network hardeningss commands (#14260)
+* Add adaptive_application_controls commands (#14278)
+* Addition of az security iot-solution/ iot-alerts/iot-recommendations/iot-analytics REST to Azure CLI (#14124)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:106]:
```suggestion
**Security**

* Add allowed_connections commands (#14190)
* Add Adaptive network hardeningss commands (#14260)
* Add adaptive_application_controls commands (#14278)
* Addition of az security iot-solution/ iot-alerts/iot-recommendations/iot-analytics REST to Azure CLI (#14124)
* Add regulatory compliance CLI (#14103)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,131 @@
...
+
+* `az storage account create/update`: Add --allow-blob-public-access to allow or disallow public access for blob and containers (#13986)
+* `az storage account create/update`: Add `--min-tls-version` to support setting the minimum TLS version to be permitted on requests to storage. (#14131)
+* Remove check in token credential (#14134)
+* Fix the storage account name in examples (#14062)
+
+**Thoth**
+
+* : recommend_recovery_options: version: "2.8.0", command: "account", parameters: "['--debug']", extension: "None" (#14180)
+* : User must agree to telemetry before failure recovery recommendations can be presented. (#14180)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:124]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,126 @@
...
+
+* Bump version disks 2020-05-01, compute 2020-06-01 (#14212)
+* Double encryption of disk encryption set (#14212)
+* `az vmss update`: support specify cross tenant image. (#14206)
+* `az sig image-version create`: support specify cross tenant image. (#14206)
+* vm/vmss create: Encryption of cache & data-in-transit for OS/Data disks and temp disks for VM & VMSS (#13919)
+
+**compute**
+
+* Add simulate-eviction operation for VM and VMSS (#14133)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:47]:
```suggestion
* vm/vmss create: Encryption of cache & data-in-transit for OS/Data disks and temp disks for VM & VMSS (#13919)
* Add simulate-eviction operation for VM and VMSS (#14133)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,123 @@
...
+
+* `az account get-access-token`: Show expiresOn for Managed Identity (#14128)
+
+**Rdbms**
+
+* Add Infrastructure Encryption for Azure Postgres and MySQL (#14097)
+
+**RDBMS**
+
+* Support Minimum TLS version (#14166)
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:92]:
```suggestion
**RDBMS**

* Support Minimum TLS version (#14166)
* Add Infrastructure Encryption for Azure Postgres and MySQL (#14097)
```

**Pull request merged**